introduction.md

```diff
@@ -0,0 +1 @@
+Hello! I am here to teach people about Github.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
You're not showing up as a collaborator anymore.  I've granted access to the Mastering Github course, but it's not working.